tests/profiles-overlays-test_test.go

```diff
@@ -21,23 +21,37 @@ bases:
 - ../../base
 #generators:
 #- app_test.yaml
+`)
```

Comment:
is the change in this file needed?

Reply:
Yes, seems this test on master head is broken